src/main/java/org/apache/commons/compress/archivers/tar/TarArchiveInputStream.java

```diff
@@ -26,14 +26,14 @@
 import java.io.ByteArrayOutputStream;
 import java.io.IOException;
 import java.io.InputStream;
-import java.util.HashMap;
-import java.util.Map;
+import java.util.*;
```

Comment:
unless rules have changed - please avoid * imports

Reply:
Agree. Sorry for my careless.

Reply:
resolved with b465bfa6